vespajlib/src/main/java/com/yahoo/slime/ArrayValue.java

```diff
@@ -10,9 +10,11 @@ final class ArrayValue extends Value {
     static final Impl initial_impl = new EmptyImpl();
 
     private interface Impl {
-        public void prepareFor(ArrayValue self, Type type);
-        public Value add(Value value, int used);
-        public Value get(int index);
+        void prepareFor(ArrayValue self, Type type);
```

Comment:
as long as the implementations are public, these should be public as well

Reply:
Interface are always public.